taqueria-flextesa-manager/lib.ts

```diff
@@ -68,7 +68,7 @@ export const importAccounts = (opts: SanitizedArgs.t) => {
 			if (accountName === 'default') return retval;
 			const account = accountDetails as SandboxAccountConfig.t;
 			// const protocol = getSandboxProtocol(opts)
-			const protocol = 'ProtoALphaAL';
+			const protocol = 'PtKathmankSp';
```

Comment:
May I suggest we put it at the top and use a const variable?

e.g. `const PROTOCOL_HASH = "PtKathmankSp"`

Reply:
Done